pilot/pkg/networking/core/v1alpha3/tracing.go

```diff
@@ -135,21 +143,14 @@ func configureFromProviderConfig(pushCtx *model.PushContext, meta *model.NodeMet
 
 	switch provider := providerCfg.Provider.(type) {
 	case *meshconfig.MeshConfig_ExtensionProvider_Zipkin:
-		tracing, err = buildHCMTracing(pushCtx, providerCfg.Name, provider.Zipkin.Service, provider.Zipkin.Port, provider.Zipkin.MaxTagLength, zipkinConfigGen)
+		tracing, err = buildHCMTracing(pushCtx, envoyZipkin, provider.Zipkin.Service, provider.Zipkin.Port, provider.Zipkin.MaxTagLength, zipkinConfigGen)
```

Comment:
any reason to change this to const value?

Reply:
the [envoy docs](https://www.envoyproxy.io/docs/envoy/v1.23.0/api-v3/config/trace/v3/http_tracer.proto#config-trace-v3-tracing-http) claim that we were setting the incorrect name: 

> name
([string](https://developers.google.com/protocol-buffers/docs/proto#scalar), REQUIRED) The name of the HTTP trace driver to instantiate. The name must match a supported HTTP trace driver. See the [extensions listed in typed_config below](https://www.envoyproxy.io/docs/envoy/v1.23.0/api-v3/config/trace/v3/http_tracer.proto#extension-category-envoy-tracers) for the default list of the HTTP trace driver.

Reply:
oh, this's different from ALS.